composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "sybio/image-workshop",
+    "name": "newerton/image-workshop",
```

Comment:
Please don't change the `composer.json` package name because it will break current packagist process.